website/client/src/assets/scss/page.scss

```diff
@@ -20,5 +20,21 @@ body {
 }
 
 .page-header {
-  color: $purple-200;
+  color: $purple-300;
+
+  height: 2rem;
+  font-size: 24px;
+  font-weight: bold;
+  font-stretch: condensed;
```

Comment:
I don't think these font-stretch properties work, the only way to have it working is using the `font-family: 'Roboto Condensed', sans-serif;` which btw comes by default with h{1,2,3,4,5,6} elements

Reply:
I think this was a chunk of a failed attempt before I used the class `textCondensed` on the same place as page-header - will be removed